### DISCUSSION TITLE
Change the IP address translation to a different IP address

### DISCUSSION THREAD

**Discussion by PanSzelescik** [AdguardTeam/AdGuardHome#6743] [General]
Hello,

Since the last time [I, and not only I](https://community.cloudflare.com/t/latency-to-proxied-dns-entries-high-since-2024-01-30t0715cet/608356/23) have been encountering large pings to CloudFlare servers.

I thought that since I was using AdGuard Home I would change my domain name translation to [a faster address](https://community.cloudflare.com/t/latency-to-proxied-dns-entries-high-since-2024-01-30t0715cet/608356/10), to which I have a lower ping (50 ms vs 150 ms).

And yes, I can do this for one domain, or several domains via Filters > DNS rewrites or Filters > Custom filtering rules, but as you know quite a lot of the internet is hosted on CloudFlare servers, so it doesn't see me typing in so many domains.

Is there therefore an option to:
- Normally the A domain record of example.com is 0.0.0.0
- I would like to set that for all domains that respond to a given IP address (0.0.0.0), I would like to change this IP to, for example, 1.1.1.1

If this option is not currently available, please treat this as a feature request
Thanks

**Comment by ainar-g** [comment]
See/[upvote](https://github.com/AdguardTeam/AdGuardHome/blob/master/CONTRIBUTING.md#general-recommendations) #6325.
